cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/LauncherUtils.java

```diff
@@ -99,23 +100,22 @@ public static void runProgram(Path sourceRootPath, Path sourcePath, String funct
             programFile = compile(fullPath.getParent(), fullPath.getFileName(), offline);
         } else if (Files.isDirectory(sourceRootPath)) {
             if (Files.isDirectory(fullPath) && !RepoUtils.hasProjectRepo(sourceRootPath)) {
-                throw LauncherUtils.createLauncherException(
-                        "error: did you mean to run the Ballerina package as a project? If so run 'ballerina init' to" +
-                                " make it a project with a .ballerina directory");
+                throw createLauncherException("did you mean to run the Ballerina package as a project? If so run "
+                                                      + "'ballerina init' to make it a project with a .ballerina "
+                                                      + "directory");
             }
             programFile = compile(sourceRootPath, sourcePath, offline);
         } else {
-            throw LauncherUtils.createLauncherException(
-                    "error: only packages, " + BLANG_SRC_FILE_SUFFIX + " and " + BLANG_EXEC_FILE_SUFFIX +
-                            " files can be used with the 'ballerina run' command.");
+            throw createLauncherException("only packages, " + BLANG_SRC_FILE_SUFFIX + " and " + BLANG_EXEC_FILE_SUFFIX
+                                                  + " files can be used with the 'ballerina run' command.");
         }
 
         // If a function named main is expected to be the entry point but such a function does not exist and there is
         // no service entry point either, throw an error
         if ((MAIN_FUNCTION_NAME.equals(functionName) && !programFile.isMainEPAvailable())
                 && !programFile.isServiceEPAvailable()) {
-            throw LauncherUtils.createLauncherException(
-                    "error: '" + programFile.getProgramFilePath() + "' does not contain a main function or a service");
+            throw createLauncherException("'" + programFile.getProgramFilePath()
+                                                  + "' does not contain a main function or a service");
```

Comment:
I think it is better to keep the `+` in the line above. Then it is easier to notice that a concatenation is happening before checking the next line. WDYT? This can be fixed in some of the changes above and below as well.

Reply:
Isn't "breaking before the operator" the standard we follow?

http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html#248

I personally prefer breaking before the operator since it is clear at first glance that such a line is a continuation of another.

Reply:
If you check the codebase, almost all of the places have used "breaking after the operator" convention. I wonder if it is because of the personal preference 🤔